src/movies.js

```diff
@@ -1,16 +1,134 @@
 // Iteration 1: All directors? - Get the array of all directors.
+function getAllDirectors(movies) {
+    let answer = movies.map(function(movie){
```

Comment:
Try to use more meaningful names for your variables, something that reflects the value that will be stored. For this iteration could be directorsArray or something like that.